### PR TITLE
Enable swish payout

### PR DESCRIPTION
**Pull request by FredrikAreschoug**
## What?
- Enable swish payout

## Why?
- :money_with_wings: 

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java [hunk 1]
```diff
@@ -297,8 +300,49 @@ public PaymentCompletionResponse createPaymentCompletionLink(final String member
...
+        return addPayment(id, claimPayment, env);
+    }
+
+    public CompletableFuture<DataFetcherResult<Claim>> createClaimSwishPayment(
+        UUID id,
+        ClaimSwishPaymentInput paymentInput,
+        DataFetchingEnvironment env
+    ) throws AuthorizationException {
+        log.info("Personnel with email '{}'' adding claim payment",
+            GraphQLConfiguration.getEmail(env, personnelService));
```
Comment (bystam) [inline src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java:319]: This places GDPR protected information (email) in our logs. Do we say explicitly that that's okay for our own employees? If not, I think it's probably something to be wary of.

File: src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java [hunk 2]
```diff
@@ -297,8 +300,49 @@ public PaymentCompletionResponse createPaymentCompletionLink(final String member
             paymentInput.getExGratia(),
             ClaimPaymentType.valueOf(paymentInput.getType().toString()),
             GraphQLConfiguration.getEmail(env, personnelService),
-            paymentInput.isSanctionListSkipped()
+            paymentInput.isSanctionListSkipped(),
+            null,
+            null,
+            null,
+            null,
```
Comment (bystam) [inline src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java:307]: How come all these nulls arrived. Is that a lombok thing that it generates all the different constructors or something? And that you had to fill it with `null`s to reach the `SelectedPayoutDetails` type?

**Review by bystam — COMMENTED** [review]
One comment and one curious question :)

File: src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java (hunk 1 above)
Comment (Michael-Duivestein) [reply src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java:319]: Does that also count when it comes to work email accounts in an office environment?

Comment (FredrikAreschoug) [reply src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java:320]: We should probably remove this in more places yes

File: src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java (hunk 2 above)
Comment (FredrikAreschoug) [reply src/main/java/com/hedvig/backoffice/graphql/GraphQLMutation.java:305]: Yeah Lombok makes java a bit nicer but yeah I had to add them to be able to add `SelectedPayoutDetails.NotSelected.INSTANCE`. One day we should convert this class!!

File: src/main/java/com/hedvig/backoffice/services/payments/PaymentServiceClient.java [hunk 3]
```diff
@@ -36,10 +36,6 @@ void chargeMember(
     @PostMapping("/v2/_/members/{memberId}/payout")
     ResponseEntity<UUID> payoutMember(
         @PathVariable String memberId,
-        @RequestParam(name = "category") PayoutCategory category,
```
Comment (AlexSJ1990) [inline src/main/java/com/hedvig/backoffice/services/payments/PaymentServiceClient.java:39]: Nice 🧹

File: src/main/java/com/hedvig/backoffice/services/claims/ClaimsServiceStub.java [hunk 4]
```diff
@@ -93,7 +95,8 @@ public ClaimsServiceStub() {
                 LocalDateTime.now(),
                 ClaimPaymentStatus.COMPLETED,
                 null,
-                "HDI"
+                "HDI",
+                SelectedPayoutDetails.NotSelected.INSTANCE
```
Comment (AlexSJ1990) [inline src/main/java/com/hedvig/backoffice/services/claims/ClaimsServiceStub.java:99]: what does the INSTANCE do?

**Review by AlexSJ1990 — APPROVED** [review]
✅

File: src/main/java/com/hedvig/backoffice/services/claims/ClaimsServiceStub.java (hunk 4 above)
Comment (FredrikAreschoug) [reply src/main/java/com/hedvig/backoffice/services/claims/ClaimsServiceStub.java:99]: In java, you need to specify that you need an instance of a kotlin object to work. To solve this kotlin create a static field called `INSTANCE`

**Pull request merged**